packages/attribution/src/attribution.ts

```diff
@@ -181,20 +183,66 @@ export class Attribution {
   isIgnored?: (attr: AttributionBounds) => boolean;
 
   /**
-   * Render the filtered attributions as a simple string suitable to display as attribution
+   * Parse the filtered list of attributions into a formatted string comprising license information.
+   *
+   * @param filtered The filtered list of attributions.
+   *
+   * @returns A formatted license string.
+   *
+   * @example
+   * if (filtered[0] contains no providers or licensors):
+   * return "CC BY 4.0 LINZ - Otago 0.3 Rural Aerial Photos (2017-2019)"
+   *
+   * @example
+   * if (filtered[0] contains licensors):
+   * return "CC BY 4.0 Otago Regional Council - Otago 0.3 Rural Aerial Photos (2017-2019)"
+   */
+  renderLicense(filtered: AttributionBounds[]): string {
+    const providers = filtered[0]?.collection.providers;
+    const attribution = createLicensorAttribution(providers);
+    const list = this.renderList(filtered);
+
+    if (list.length) {
```

Comment:
its good being super explicit with if statements in javascript.

```suggestion
    if (list.length > 0) {
```